### DISCUSSION TITLE
Bootstrap v5

### DISCUSSION THREAD

**Discussion by vitalijalbu** [reactstrap/reactstrap#2247] [General]
Hi everyone, I was wondering if there will be a reactstrap v5 bootstrap.

**Comment by bestguy** [comment]
Hi @vitalijalbu , 
Yes, there's currently a beta release `reactstrap@9.0.0-2` which supports latest Bootstrap 5.1.
Please see this issue:
https://github.com/reactstrap/reactstrap/issues/1748#issuecomment-907471312

Only thing pending is some doc updates to support v5 & v4 docs.  
PR is currently in progress and I think Reactstrap 9 can be released soon.

**Reply by vitalijalbu** [reply]
We can see some progress or releases? Because I saw react-bootstrap already switched to v5.

**Reply by bestguy** [reply]
Hi, yes my comment above and attached thread explains where we are at.  I think this week or early next should be official release.  The latest 9.x beta is feature complete if would like to try.

**Comment by kwiato** [comment]
Are there plans to support Bootstrap v5.2?
